scripts/utilities/check_glibc_disable_crypt.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright 2017 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Check if it´s needed to compile glibc´s libcrypt.
+# If libxcrypt is available in OS, glibc´s libcrypt will not be compiled.
+# Except in cross build. In this case, glibc's libcrypt must be  compiled because it´s required to build gcc
```

Comment:
This comment should be broken in two lines.

Reply:
changes made.